packages/components/input/__tests__/input.test.tsx

```diff
@@ -116,4 +117,32 @@ describe("Input", () => {
 
     expect(ref.current?.value)?.toBe(value);
   });
+
+  it("should clear the value and onClear is triggered", async () => {
+    const onClear = jest.fn();
+
+    const ref = React.createRef<HTMLInputElement>();
+
+    const {container} = render(
+      <Input
+        ref={ref}
+        isClearable
+        defaultValue="junior@nextui.org"
+        label="test input"
+        onClear={onClear}
+      />,
+    );
+
+    const clearButton = container.querySelector("[data-slot='clear-button']") as HTMLElement;
+
+    expect(clearButton).not.toBeNull();
+
+    await act(async () => {
+      await userEvent.click(clearButton);
+    });
```

Comment:
As recommended in the documentation:
> This exists to ease the transition to version 14 and writing simple tests. Using the methods on the instances returned by userEvent.setup() is recommended.

https://testing-library.com/docs/user-event/setup/#direct-apis

```suggestion
    const user = userEvent.setup();

    await user.click(clearButton);
```